src/topology/category/Compactum.lean

```diff
@@ -0,0 +1,467 @@
+/-
+Copyright (c) 2020 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Adam Topaz
+-/
+
+import category_theory.monad.types
+import category_theory.monad.limits
+import category_theory.equivalence
+import topology.category.Top
+import data.set.constructions
+
+/-!
+
+# Compact Hausdorff Spaces
+
+This file proves the equivalence between the category of compact Hausdorff topological spaces
+and the category of algebras for the ultrafilter monad.
```

Comment:
It would be nice to explain the meaning of this sentence. We have mathematical explanations in module docstrings for much more common gadgets (such as filters or uniform spaces). Or is this sentence meant to tell readers that they don't care about this file if they don't know about ultrafilter monads? Maybe a better question is: is this file meant to be used later on, or only as a nice experiment playing with category theory and topology? If there is no intended use then it can easily stay cryptic.

Reply:
I added a more detailed discussion in the documentation of `Compactum.lean`.